tsconfig.json

```diff
@@ -18,7 +18,8 @@
       "DOM.Iterable"
     ],
     "rootDir": "src",
-    "outDir": "./"
+    "outDir": "./",
+    "resolveJsonModule": true
```

Comment:
Do we need it? 
If that's tests, we should limit that option to tests only.
Json resolving is not a native future for both ES Modules (browser) and bundlers, so if TS allows and leaves it in output - it is a problem.